src/meta/ActiveHostsMan.cpp

```diff
@@ -94,8 +99,11 @@ void ActiveHostsMan::loadHostMap() {
         HostInfo info;
         info.lastHBTimeInSec_ = time::TimeUtils::nowInSeconds();
         if (iter->val() == MetaServiceUtils::hostValOnline()) {
```

Comment:
I think we need to set status of hosts "which is online in kvstore but not in memory" to offline? Because the info in memory is more updated. After leader really receive heartbeat, it will set info to online.